docs/index.md

```diff
@@ -1,19 +1,18 @@
-# Replace This Title!
+# Project Easter Rising
 
 ---
 
-![Replace Me, Sample Image](docs/files/caesarian_code.png)
+![Finean Brotherhood. Sourced from http://irishhistoryreview.com/wp/2017/03/29/america-and-the-easter-rising/](docs\files\easterrisingmarch.jpg)
```

Comment:
the path for the image should be changed to foward slashes.... docs/files/easterrisingmarch.jpg. Otherwise it will not load correctly.